index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+
+    <!-- <header> -->
+      <nav id="navbar">
+        <a class="nav" href="#" >  About</a>
+        <a  class="nav" id="profile-link"  href="https://github.com/Azonkeu" target="_blank">  Work</a>
+        <a class="nav" href="#" >  Contact</a>
```

Comment:
Please use a `ul` with `li`. It is a great way to  organize your site's navigation and the semantic markup will improve accessibility for users that require screen-readers.